scripts/happy

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
again this... should be fine... if people are using venvs and/or global python defaults appropriately -- we don't have anything running on Python 2, right?

Reply:
I would roll back this change. It will break for people who have their default `python` command set to python 2.7.

Reply:
I'm guessing that `happy` wouldn't even run on Python 2.

Reply:
given that this uses happy requirements, this should only ever be executed when in an active venv, no?